bin/prep-release.php

```diff
@@ -49,6 +48,8 @@ function get_files() {
         "src/contrib/*.{c,h}",
 
         "src/libmongoc/src/common/src/*.{c,h,h.in}",
+        // Note: src/libmongoc/src/common/src/mlib/ does not contain source files (as of libmongoc 2.0.1)
+        "src/libmongoc/src/common/src/mlib/*.{c,h}",
```

Comment:
```suggestion
        "src/libmongoc/src/common/src/mlib/*.h",
```

Remove unnecessary `*.c` glob? I expect the purpose of the comment is to indicate there are no `*.c` source files needed.

Reply:
Updated - these are indeed not necessary (here and below).